lmdeploy/pytorch/backends/cuda/graph_runner.py

```diff
@@ -128,6 +133,14 @@ def _fill_inputs(self, input_ids: torch.Tensor, position_ids: torch.Tensor,
                 self.input_buffers['inputs_embeds'] = inputs_embeds.new_zeros(
                     1, max_num_tokens, emb_size)
             self.input_buffers['inputs_embeds'][:, :num_tokens] = inputs_embeds
+        if mrope_position_ids is not None:
```

Comment:
Is mrope_position_ids always exist?

Reply:
For qwen2-vl model, it will always exist.

Reply:
> For qwen2-vl model, it will always exist.

If there is no image input, it can be None. Just raised an error if you use `lmdeploy chat` command. However, pipline worked even if there is no image input.